tools/updateBootstrapAccessibilityPlugin.R

```diff
@@ -8,26 +8,9 @@ library(rprojroot)
 
 sha   <- "fbbf870eafc1ee5d4547fabbeea71778ddbe2166"
 dest_dir  <- rprojroot::find_package_root_file("inst/www/shared/bootstrap/accessibility")
+unlink(dest_dir, recursive = TRUE)
+dir.create(dest_dir)
 
-min_js_url       <- sprintf("https://raw.githubusercontent.com/paypal/bootstrap-accessibility-plugin/%s/plugins/js/bootstrap-accessibility.min.js", sha)
-css_url       <- sprintf("https://raw.githubusercontent.com/paypal/bootstrap-accessibility-plugin/%s/plugins/css/bootstrap-accessibility.css", sha)
-url <- c(min_js_url, css_url)
 
-# Downloading each required file:
-lapply(url, function(x) {download.file(x, file.path(tempdir(), basename(x)))})
-
-# Copying js files:
-dir.create(file.path(dest_dir, "js"), recursive = TRUE)
-file.copy(
-  file.path(tempdir(), "bootstrap-accessibility.min.js"),
-  file.path(dest_dir, "js"),
-  overwrite = TRUE
-)
-
-# Copying css file:
-dir.create(file.path(dest_dir, "css"), recursive = TRUE)
-file.copy(
-  file.path(tempdir(), "bootstrap-accessibility.css"),
-  file.path(dest_dir, "css"),
-  overwrite = TRUE
-)
+url <- sprintf("https://raw.githubusercontent.com/paypal/bootstrap-accessibility-plugin/%s/plugins/js/bootstrap-accessibility.min.js", sha)
```

Comment:
The content of the JS file should be wrapped in a `$()` so that it runs on load. (We'll need a patch for this)